app/services/participant_outcomes/create.rb

```diff
@@ -0,0 +1,85 @@
+# frozen_string_literal: true
+
+module ParticipantOutcomes
```

Comment:
@leandroalemao this is already implemented in https://github.com/DFE-Digital/npq-registration/pull/1498/files#diff-f633d59f99e20b9c30469f9dff5e59e7612da3eca47b2055f7b9e1b05004635dR4 could we please hold off on this pr till that gets merged or branch off that one please?

Reply:
Putting this PR on hold for now till we get Ross' PR merged. Tks for flagging up @cwrw